app/javascript/packs/public.js

```diff
@@ -3,6 +3,7 @@ import { getLocale } from 'mastodon/locales';
 import { length } from 'stringz';
 import IntlRelativeFormat from 'intl-relativeformat';
 import { delegate } from 'rails-ujs';
+import loadPolyfills from '../mastodon/load_polyfills';
```

Comment:
Actually TBH I find this a bit annoying because my editor (WebStorm) no longer allows me to ctrl-click to jump to the declaration for the imported module. I also can't find where the `mastodon` alias is declared. I don't think removing three characters is worth it.